module-services/service-desktop/ServiceDesktop.cpp

```diff
@@ -52,7 +57,7 @@ sys::ReturnCodes ServiceDesktop::InitHandler()
     connectHandler<sdesktop::usb::USBDisconnected>();
     connectHandler<sevm::USBPlugEvent>();
 
-    checkChargingCondition();
+    checkChargingConditionTimer.start();
```

Comment:
Please consider instead of the initialization delay the response delay in OS -> Center communication

Reply:
Good idea, I will check it